backend/api/views.py

```diff
@@ -575,3 +575,26 @@ def delete(self, request, *args, **kwargs):
 			status=status.HTTP_200_OK
 		)
 
+class ResourceView(APIView):
+	permission_classes = [IsAdminUser]
+	
```

Comment:
permission_classes isn't used anywhere

Reply:
That's how [permissions](https://www.django-rest-framework.org/api-guide/permissions/) are handled in Django Rest Framework.